### PR TITLE
2.7 release notes

### PR DESCRIPTION
**Pull request by Gallaecio**

### CODE REVIEW

File: docs/news.rst [hunk 1]
```diff
@@ -3,6 +3,187 @@
...
+-   :class:`~scrapy.spiders.crawl.CrawlSpider` now supports :ref:`asynchronous
+    callbacks <topics-coroutines>` (:issue:`5657`).
+
+-   New projects created with the :command:`startproject` command have
+    :ref:`asyncio support <using-asyncio>` enabled by default (:issue:`5590`,
+    :issue:`5679`).
+
+-   The :setting:`FEED_EXPORT_FIELDS` setting can now be defiend as a
+    dictionary to customize the output name of item fields, allowing them to
+    include non-ASCII characters (:issue:`1008`, :issue:`3266`, :issue:`3696`).
```
Comment (kmike) [inline docs/news.rst:68]: Python also allows non-ASCII characters as attribute names. After the change output names can include whitesace, and don't need to be valid Python identifiers.

File: docs/news.rst [hunk 2]
```diff
@@ -3,6 +3,187 @@
...
+
+-   The :command:`parse` command now supports :ref:`asynchronous callbacks
+    <topics-coroutines>` (:issue:`5424`, :issue:`5577`).
+
+-   When using the :command:`parse` command with a URL for which there is no
+    available spider, an exception is no longer raised (:issue:`3264`,
+    :issue:`3265`, :issue:`5375`, :issue:`5376`, :issue:`5497`).
+
+-   :class:`~scrapy.http.TextResponse` now takes into account the `byte order
+    mark`_ when determining the text encoding of the response body
```
Comment (kmike) [inline docs/news.rst:102]: It was taking BOM in account before Scrapy 2.7, but not doing it correctly;

File: docs/news.rst [hunk 3]
```diff
@@ -3,6 +3,187 @@
...
+    (:issue:`5601`, :issue:`5611`).
+
+    .. _byte order mark: https://en.wikipedia.org/wiki/Byte_order_mark
+
+-   MIME sniffing can now detect HTML 5 responses based on their body, and
```
Comment (kmike) [inline docs/news.rst:107]: It reads as there was no MIME sniffing based on response body; if I'm not mistaken, the point here is that it's now working for cached, HTTP/1.0 and FTP responses.

File: docs/news.rst (hunk 1 above)
Comment (Gallaecio) [reply docs/news.rst:68]: Today I learned…

File: docs/news.rst (hunk 3 above)
Comment (Gallaecio) [reply docs/news.rst]: I was overestimating the extent of the other part of the fix, looking into it further I see the only fix was in the detection of HTML 5 documents *without* the `html` tag.

File: docs/news.rst [hunk 4]
```diff
@@ -3,6 +3,192 @@
...
+
+-   :class:`~scrapy.spiders.crawl.CrawlSpider` now supports :ref:`asynchronous
+    callbacks <topics-coroutines>` (:issue:`5657`).
+
+-   New projects created with the :command:`startproject` command have
+    :ref:`asyncio support <using-asyncio>` enabled by default (:issue:`5590`,
+    :issue:`5679`).
+
+-   The :setting:`FEED_EXPORT_FIELDS` setting can now be defiend as a
+    dictionary to customize the output name of item fields, lifting the
```
Comment (kmike) [inline docs/news.rst:67]: I think we should add it to highlights.

File: docs/news.rst [hunk 5]
```diff
@@ -3,6 +3,192 @@
...
+-   ``jsonl`` is now supported and encouraged as a file extension for `JSON
+    Lines`_ files (:issue:`4848`).
+
+    .. _JSON Lines: https://jsonlines.org/
+
+-   :meth:`ImagesPipeline.thumb_path
+    <scrapy.pipelines.images.ImagesPipeline.thumb_path>` now receives the
+    source :ref:`item <topics-items>` (:issue:`5504`, :issue:`5508`).
+
+-   You can now customize :ref:`request fingerprinting <request-fingerprints>`
```
Comment (kmike) [inline docs/news.rst:81]: This might also get it into the highlights.

**Review by kmike — APPROVED** [review]
Looks great, thanks @Gallaecio!

File: docs/news.rst [hunk 6]
```diff
@@ -3,6 +3,195 @@
...
+    (:issue:`4978`).
+
+-   :class:`~scrapy.spiders.crawl.CrawlSpider` now supports :ref:`asynchronous
+    callbacks <topics-coroutines>` (:issue:`5657`).
+
+-   New projects created with the :command:`startproject` command have
+    :ref:`asyncio support <using-asyncio>` enabled by default (:issue:`5590`,
+    :issue:`5679`).
+
+-   The :setting:`FEED_EXPORT_FIELDS` setting can now be defiend as a
```
Comment (wRAR) [inline docs/news.rst:69]: Typo, "defiend"

Comment (Gallaecio) [reply docs/news.rst]: And I made a typo on the commit message to fix the typo…

**Review by wRAR — APPROVED** [review]

File: docs/news.rst (hunk 6 above)
Comment (wRAR) [reply docs/news.rst]: I thought it was intentional!

**Pull request merged**